webapp/utils/token.ts

```diff
@@ -56,3 +56,6 @@ export const getErc20TokenBalance = ({
     args: [address],
     functionName: 'balanceOf',
   })
+
+export const isStakeToken = (token: Token) =>
+  token.extensions?.protocol !== undefined
```

Comment:
Why thats is better than this?

```suggestion
export const isStakeToken = (token: Token) =>
  !!token.extensions?.protocol
```

Reply:
It's not, it's just different. Sometimes I find it more readable because I don't have to think about falsy/truthy values, but it's not that "it's better". 🤷🏽